src/graphql/fragments/loanCardFields.graphql

```diff
@@ -4,6 +4,7 @@ fragment loanCardFields on LoanBasic {
 	name
 	borrowerCount
 	geocode {
+		city
```

Comment:
My understanding was I needed to add `city` here if I wanted to access it from the GQL cache

Reply:
Since this fragment is used in a query for fetching the loans for the category rows, this will indeed populate the cache with that value. However, as we build more of this loan card we will probably want separate queries so that we only fetch the extra info we need for the card when we're displaying it

Reply:
Agree. We're currently fetching a lot of unused information for the initial rows with these tiny loan cards.